lib/code_corps_web/controllers/github_app_installation_controller.ex

```diff
@@ -1,29 +1,49 @@
 defmodule CodeCorpsWeb.GithubAppInstallationController do
   use CodeCorpsWeb, :controller
-  use JaResource
 
   import CodeCorps.Helpers.Query, only: [id_filter: 2]
 
-  alias CodeCorps.{GithubAppInstallation}
+  alias CodeCorps.{GithubAppInstallation, User}
 
-  @preloads [:project, :user]
-
-  plug :load_resource, model: GithubAppInstallation, only: [:show], preload: @preloads
-  plug :load_and_authorize_changeset, model: GithubAppInstallation, only: [:create], preload: @preloads
-
-  plug JaResource
+  action_fallback CodeCorpsWeb.FallbackController
+  plug CodeCorpsWeb.Plug.DataToAttributes
+  plug CodeCorpsWeb.Plug.IdsToIntegers
 
   @spec model :: module
   def model, do: CodeCorps.GithubAppInstallation
```

Comment:
You can completely remove the `model` function. It's used internally by `ja_resource` only.

Reply:
thx